xrdp/xrdp_wm.c

```diff
@@ -1240,6 +1240,11 @@ xrdp_wm_mouse_click(struct xrdp_wm *self, int x, int y, int but, int down)
         {
             if (self->mm->mod->mod_event != 0)
             {
+				if (down)
```

Comment:
The formatting looks wrong. Please don't use tabs, it's described in coding_style.md